src/WooCommerce.php

```diff
@@ -515,13 +518,21 @@ public static function woocommerce_get_item_data($data, $cartItem) {
   public static function woocommerce_display_item_meta($html, $item, $args) {
     $strings = [];
     $product = $item->get_product();
-    $attributes = array_merge(static::getProductData($product), static::getProductAttributes($product));
-    foreach ($attributes as $attribute) {
-      $strings[] = '<strong class="wc-item-meta-label">' . $attribute['name'] . ':</strong> ' . $attribute['value'];
+    $product_data_set = array_merge(static::getProductData($product), static::getProductAttributes($product));
+
+    if ($delivery_time = get_term(wc_get_order_item_meta($item->get_id(), '_deliverytime'), 'product_delivery_times')->name) {
+      array_splice($product_data_set, 1, 0, [[
```

Comment:
Also wondering whether this part duplicates the other PR that Luca already mentioned.

Reply:
I already cancelled/closed that PR. Duplicated stuff, sorry.